src/pyJiraCli/cmd_import.py

```diff
@@ -133,7 +133,17 @@ def _create_components(jira: JIRA, components: list[dict], project_key: str) ->
 
     ret_status = Ret.CODE.RET_OK
 
+    # Get the existing components of the project.
+    existing_components = jira.project_components(project_key)
+
     for component in components:
+        # Check if the component already exists.
+        if any((component.get("name") == existing_component.name)
+                for existing_component in existing_components):
```

Comment:
Please move the check with any() to a separate result variable and check the result variable.